samples/AnimatedModel3DExample/Program.cs

```diff
@@ -0,0 +1,408 @@
+﻿using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Presentation;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace AnimatedModel3DExample
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            if (args.Length < 3)
```

Comment:
Can you make a common file for this stuff that we can include in all the samples?